nilearn/plotting/surface/tests/test_surf_plotting.py

```diff
@@ -131,7 +131,6 @@ def test_plot_surf(plt, engine, tmp_path, in_memory_mesh, bg_map):
 
     # Plot mesh with background
     plot_surf(in_memory_mesh, bg_map=bg_map, engine=engine)
-    plot_surf(in_memory_mesh, bg_map=bg_map, darkness=0.5, engine=engine)
```

Comment:
I guess this is removed since we're deprecating `darkness` ?

Reply:
yes exactly